.azure-pipelines/scripts/run_test.sh

```diff
@@ -21,7 +21,7 @@ fi
 tests_dir=$SGXLKL_ROOT/tests
 test_name="$(realpath --relative-to="$tests_dir" "$(pwd)")"
 test_name="${test_name//\//-}"
-test_name+="-($build_mode)-($run_mode)"
+test_name+="-($build_mode)-($run_mode)-($SGXLKL_ETHREADS)"
```

Comment:
This will just a number to the test name. Adding also "threads" suffix will be more clear.